cfgov/jinja2/v1/_includes/conference-registration-form.html

```diff
@@ -0,0 +1,107 @@
+{% import 'atoms/input.html' as input %}
+{% import 'atoms/textarea.html' as textarea %}
+{% import 'atoms/submit-button.html' as submit_button %}
+{% import 'molecules/big-checkbox.html' as big_checkbox %}
+{% import 'molecules/notification.html' as notification %}
+
+{% macro render(value, settings, form_id) %}
+{% set form = settings.form %}
+
+<form id="registration-form"
+      method="post"
+      action="."
+      class="o-form js-validate-filters">
+
+    <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
```

Comment:
This was perhaps something that should have been discussed in #2411, but this will not work with Akamai, because the token will be cached and so will be the same for every form submission. I believe this is the reason why `@csrf_exempt` is required [here](https://github.com/cfpb/cfgov-refresh/blob/e0e0f874adb344f5afdd47a6404a7acdb5004478/cfgov/core/views.py#L62).